test/Interop/Cxx/reference/Inputs/reference-silgen-cxx-objc-ctors+init.h

```diff
@@ -0,0 +1,18 @@
+
+
+@interface NSObject
+@end
+
+struct PrimitiveWrapperT { int value; };
+
+struct CxxEmbeddedT {
+  PrimitiveWrapperT V;
+  CxxEmbeddedT() = delete;
+  CxxEmbeddedT(const PrimitiveWrapperT &V) noexcept : V(V) {}
+};
+
+
+@interface ObjCSwiftBridgeT : NSObject
+- (instancetype)init  __attribute__((unavailable));
+- (instancetype)initWithEmbedded:(const CxxEmbeddedT &)embedded __attribute__((objc_designated_initializer));
```

Comment:
Is all of this needed? We can't just take a `PrimitiveWrapperT` here?

Reply:
I think I was trying to see if adding an extra layer of wrapper effects if it is passed by value or as a @in*

Reply:
I remember now, for the `FIXME`

I wanted to show that the call to  `_ZN17IntWrapperWrapperC1ERK10IntWrapper` is a

` function_ref @_ZN17IntWrapperWrapperC1ERK10IntWrapper : $@convention(c) (IntWrapper)`

and not a 

` function_ref @_ZN17IntWrapperWrapperC1ERK10IntWrapper : $@convention(c) (@in_guaranteed  IntWrapper)`